wiki/docs/Home.md

```diff
@@ -22,4 +22,14 @@ When you design and implement an interface, please keep in mind following practi
 * [L4 Kernel Reference Manual](http://www.l4ka.org/l4ka/l4-x2-r7.pdf) Version X.2
 * [Linux Kernel Cross Reference](http://lxr.free-electrons.com/)
 * [FreeBSD Kernel Cross Reference](http://fxr.watson.org/fxr/source/)
-* [Minix 3.3.0 Cross Reference](http://users.sosdg.org/~qiyong/mxr/source/minix/kernel/)
\ No newline at end of file
+* [Minix 3.3.0 Cross Reference](http://users.sosdg.org/~qiyong/mxr/source/minix/kernel/)
+
+### Documentation
+
+Useful sites:
+* [OSDev wiki](http://wiki.osdev.org)
+
+Toolchain documentation:
+* [Extensions to the C Language Family](https://gcc.gnu.org/onlinedocs/gcc/C-Extensions.html)
+* [Debugging with GDB](https://sourceware.org/gdb/onlinedocs/gdb/index.html)
+* [Linker scripts](https://sourceware.org/binutils/docs/ld/Scripts.html)
```

Comment:
General comment to this file: is the first sentence on the page necessary? I remember @cahirwpz telling me it's unimportant, so why keep it there? Also, I imagine that most people that come to read wiki do it out of curiosity, so starting the home page with "Before you pick up any task [...]" seems strange to me. Shouldn't it begin with some very brief overview of what Mimiker is and what can be found in the wiki?

Reply:
Let's target this issue later. I don't want to grow this PR indefinitely.